tools/scrimlet/softnpu-init.sh

```diff
@@ -27,7 +27,24 @@ fi
 
 #   $ arp -a | grep 192.168.21.1
 #   e1000g1 192.168.21.1         255.255.255.255          90:ec:77:2e:70:27
-GATEWAY_MAC=${GATEWAY_MAC:=$(arp -a | grep "$GATEWAY_IP" | awk -F ' ' '{print $NF}')}
+#
+# Add an extrac space at the end of the search pattern passed to `grep`, so that
```

Comment:
```suggestion
# Add an extra space at the end of the search pattern passed to `grep`, so that
```

But see comment above -- should this be a separate PR?